boards/beagle/pocketbeagle_2/pocketbeagle_2_am6232_a53.dts

```diff
@@ -34,7 +34,17 @@
 	};
 
 	ddr0: memory@80000000 {
+		device_type = "memory";
 		reg = <0x80000000 DT_SIZE_M(512)>;
+		#address-cells = <0x1>;
+		#size-cells = <0x1>;
+
+		/* Loading and starting code at RAM start will cause any calls to
+		 * Arm Trusted Firmware-A to fail.
+		 */
+		code_partition: memory@80200000 {
```

Comment:
`code_partition` is used in a couple other spots but seems to always be inside some flash storage. This is DRAM, instead do as we do in Linux, mark the first couple MB as reserved, and give this a name that doesn't include "partition", maybe `a53_ddr_section`.